apps/frontend/src/components/LobbyPage.tsx

```diff
@@ -0,0 +1,7 @@
+import { Box } from '@mui/material';
+
+function LoginPage() {
```

Comment:
should be `LobbyPage`, but what is this page for?

Reply:
I imagine we would have a lobby page for the 2 users to read the question or chat, or maybe other functions before actually jumping into the interview, in the meantime im using lobby as a placeholder name